app/models/solidus_viabill/gateway.rb

```diff
@@ -91,12 +91,6 @@ def void(order_number, gateway_options)
 
     def purchase(float_amount, payment_source, gateway_options)
       capture(float_amount, payment_source.order_number, gateway_options)
```

Comment:
You will have to change the tests also if you remove this response from here

Reply:
I ran the tests and they are all successful
![image](https://user-images.githubusercontent.com/49295880/153337545-ce30412a-25a3-4050-bad3-fb0ac7cb7b15.png)

Reply:
This test on `gateway_spec.rb` should have ideally failed with the changes made.
```rb
  describe '#purchase' do
    subject(:purchase_response) { gateway.purchase(100, payment_source, { originator: payment, currency: 'USD' }) }

    before do
      response = Net::HTTPNoContent.new('', '204', 'NoContent')
      allow(gateway).to receive(:send_post_request).and_return(response)
    end

    it 'successfully returns a response' do
      expect(purchase_response.class).to eq ActiveMerchant::Billing::Response
    end
  end
```

If it's faulty can you please fix this test also ?

Reply:
why should it have failed? you are just checking that an instance of `ActiveMerchant::Billing::Response` gets created and that still happens

Reply:
that's what I'm reporting in https://github.com/nebulab/solidus_viabill/issues/7 - you used to create 2 instances but with this change you create only 1
tests are passing because the instance still gets created

Reply:
Ohh ok
Sorry, I forgot `#capture` would also return it's own instance of `ActiveMerchant::Billing::Response`.

Looks good.